lib/std/zig/BuiltinFn.zig

```diff
@@ -263,6 +264,14 @@ pub const list = list: {
                 .illegal_outside_function = true,
             },
         },
+        .{
+            "@disableInstrumentation",
```

Comment:
Should we add this built-in to the langref?

Reply:
i think it makes sense to leave out of the langref but include in the spec

Reply:
> Should we add this built-in to the langref?

Yes of course

> i think it makes sense to leave out of the langref but include in the spec

This not Meghan's Opinion Tracker. Provide technical arguments, or keep your opinion to yourself.